src/js/engine/engine.js

```diff
@@ -68,7 +74,7 @@ const openFile = async (sender, filePath) => {
       filePath,
       fileSize,
       endIndex,
-      history,
+      replaceEmptyLinesWithHiddenChar(history),
```

Comment:
Some time in the future we might have forgotten why this function call exists. (The actual problem appears to be a frontend problem solved in the backend). Maybe just add a comment why the rendering needs something else than empty string to work?